### PR TITLE
Feature form

### PR DESCRIPTION
**Pull request by azy64**
This pull request is about creating forms for Post, Comment, and Like Entities

### CODE REVIEW

File: db/migrate/20220127173633_add_users_to_posts.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddUsersToPosts < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :posts, :user, null: false, foreign_key: true
+  end
+end
```
Comment (DeliceLydia) [inline db/migrate/20220127173633_add_users_to_posts.rb]: - [ ] There is an issue with a user foreign key that is not available in `posts` table,  please fix that
fr

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @azy64, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Major highlights for your project are:
- Followed git workflow
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220127173633_add_users_to_posts.rb (hunk 1 above)
Comment (azy64) [reply db/migrate/20220127173633_add_users_to_posts.rb]: you need to remove all the migrations but not the schema

**Review by dasileker — CHANGES_REQUESTED** [review]
## **Hi @azy64,** 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- The previews colleague did not say that you need to remove the migrations files, she asked you to fix the missing `foreign_key` in your `remoe_user_from_post table`. so please revert back all the migration files and add the missing t.reference for = foreign_key to the remoe_user_from_post table.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by azy64** [comment]
in my local repository there's not error!

**Review by dasileker — APPROVED** [review]
## **Hi @azy64,** 

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**